exporter.go

```diff
@@ -1137,6 +1237,109 @@ func getKeyInfo(c redis.Conn, key string) (info keyInfo, err error) {
 	return info, err
 }
 
+func getStreamInfo(c redis.Conn, key string) (streamInfo, error) {
```

Comment:
I wanted to provide  a more specific error to the user, because `XINFO` err not so clear with types:
```
10.40.0.8:6379> XINFO STREAM test_key
(error) WRONGTYPE Operation against a key holding the wrong kind of value
```
But you are right, this info is useless: if wrong key is set (not a stream) theres is no help anyway and user must specify valid key manually.